tests/foreman/cli/test_template.py

```diff
@@ -115,7 +115,7 @@ def test_add_operating_system_1(self):
 
     @run_only_on('sat')
     @tier2
-    def test_remove_operating_system_1(self):
+    def test_positive_remove_os_by_id(self):
         """@Test: Check if OS can be removed Template
```

Comment:
I know it is out of scope, but I can't pass by:
`Check if operating system can be removed from template by id`